store/copr/coprocessor_test.go

```diff
@@ -299,6 +302,78 @@ func TestBuildTasksByBuckets(t *testing.T) {
 	for i, task := range tasks {
 		taskEqual(t, task, regionIDs[1], regionIDs[1], expectedTaskRanges[i]...)
 	}
+
+	// cover the whole region
+	// region:    n--------------x
+	// buckets:   n -- q -- r -- x
+	// ranges:    n--------------x
+	// tasks:     o -- q
+	//                 q -- r
+	//						r -- x
+	expectedTaskRanges = [][]string{
+		{"n", "q"},
+		{"q", "r"},
+		{"r", "x"},
+	}
+	cluster.SplitRegionBuckets(regionIDs[1], [][]byte{{'n'}, {'q'}, {'r'}, {'x'}}, regionIDs[1])
+	cache = NewRegionCache(tikv.NewRegionCache(pdCli))
+	defer cache.Close()
+	tasks, err = buildCopTasks(bo, cache, buildCopRanges("n", "q", "q", "r", "r", "x"), req, nil)
+	require.NoError(t, err)
+	require.Len(t, tasks, len(expectedTaskRanges))
+	for i, task := range tasks {
+		taskEqual(t, task, regionIDs[1], regionIDs[1], expectedTaskRanges[i]...)
+	}
+
+	// cross several  buckets
```

Comment:
```suggestion
	// cross several buckets
```

Reply:
It's better to move this test to the place after 'cross bucket ranges' because they are related.